prow/external-plugins/cherrypicker/server.go

```diff
@@ -492,3 +509,27 @@ func (s *Server) getPatch(org, repo, targetBranch string, num int) (string, erro
 func normalize(input string) string {
 	return strings.Replace(input, "/", "-", -1)
 }
+
+func parseCommentArguments(args string) (map[string]string, error) {
```

Comment:
GitHub ate my comment .... but lets use `strings.Contains()` to check for `#` and then for `/` and not use regex here. My sample impl for you was 10 lines